runtime/src/lib.rs

```diff
@@ -309,13 +307,13 @@ impl pallet_ethereum::Config for Runtime {
 }
 
 parameter_types! {
-	pub const BlocksPerRound: u32 = 5;
+	pub const BlocksPerRound: u32 = 20;
```

Comment:
Maybe adding a short explanation of each number here would be helpful. Like:
Moonbeam will perform a new round every 2 minutes (20 * block_time)